modules/ROOT/pages/available-menu-items.adoc

```diff
@@ -155,26 +155,12 @@ include::partial$misc/plugin-menu-item-id-boilerplate.adoc[]
 :altplugincode: nil
 include::partial$misc/plugin-menu-item-id-boilerplate.adoc[]
 
-:plugincategory: opensource
-:pluginname: Paste
-:plugincode: paste
-:altplugincode: nil
-include::partial$misc/plugin-menu-item-id-boilerplate.adoc[]
-
 :plugincategory: premium
 :pluginname: Permanent Pen
 :plugincode: permanentpen
 :altplugincode: nil
 include::partial$misc/plugin-menu-item-id-boilerplate.adoc[]
 
-:plugincategory: premium
```

Comment:
Checking this too?

Reply:
See the previous comment... the UI components are no longer in PowerPaste.